forage-android/src/main/java/com/joinforage/forage/android/collect/BTPinCollector.kt

```diff
@@ -55,12 +59,12 @@ internal class BTPinCollector(
                     )
                 )
             } catch (e: JSONException) { }
+            logger.i("Received successful response from BasisTheory")
```

Comment:
include `merchantAccount`,  `paymentMethodRef` and `cardToken` in log?

Reply:
https://github.com/teamforage/forage-android-sdk/pull/42#discussion_r1274193522

should omit cardToken